.github/workflows/tests.yml

```diff
@@ -17,29 +17,25 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v1
         with:
-          node-version: '12.x'
+          node-version: '14.x'
       - name: Install dependencies
         run: yarn --dev
       - name: Run style check
         run: yarn lint
   cypress-run:
     runs-on: ubuntu-latest
     container: cypress/browsers:node12.18.3-chrome87-ff82
-    services:
-      meilisearch:
-        image: getmeili/meilisearch:v0.24.0
-        env:
-          MEILI_MASTER_KEY: 'masterKey'
-          MEILI_NO_ANALYTICS: 'true'
-        ports:
-          - '7700:7700'
+    strategy:
+      fail-fast: false
+      matrix:
+        containers: [1, 2]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Setup node
         uses: actions/setup-node@v1
         with:
-          node-version: '12.x'
+          node-version: '14.x'
```

Comment:
I started this PR before upgrading every repository to Node v16, so I just safely upgraded to Node 14 in the meantime. 
Then I thought I would make the update to v16 in another PR because this one is already big, but yeah I'm going to do it all here, that way it will be done for good 😀